File: app/src/main/java/com/my/televip/virtuals/messenger/Utilities.java
package com.my.televip.virtuals.messenger;

import com.my.televip.Class.ClassNames;
import com.my.televip.Class.ClassLoad;
import com.my.televip.obfuscate.ArgsResolver;
import com.my.televip.obfuscate.Obfuscate;

import de.robv.android.xposed.XposedHelpers;

public class Utilities {

    public static DispatchQueue getStageQueue(){
        return new DispatchQueue(XposedHelpers.getStaticObjectField(ClassLoad.getClass(ClassNames.UTILITIES), Obfuscate.getFieldName("Utilities", "stageQueue")));
    }
}

File: app/src/main/java/com/my/televip/virtuals/messenger/Utilities.kt
package com.my.televip.virtuals.messenger

import com.my.televip.clazz.ClassLoad
import com.my.televip.clazz.ClassNames
import com.my.televip.obfuscate.Obfuscate
import de.robv.android.xposed.XposedHelpers

object Utilities {

    @JvmStatic
    fun getStageQueue(): DispatchQueue {
        val utilClass = ClassLoad.getClass(ClassNames.UTILITIES)
        val queueObj = XposedHelpers.getStaticObjectField(utilClass, Obfuscate.getFieldName("Utilities", "stageQueue"))
        return DispatchQueue(queueObj)
    }
}
